package.json

```diff
@@ -32,10 +32,11 @@
   },
   "devDependencies": {
     "chai": "^4.1.0",
+    "intercept-stdout": "^0.1.2",
```

Comment:
Why is this needed?

Reply:
This was a failed experiment and needs to be removed.